### DISCUSSION TITLE
Issues with Integrating Zitadel with Windows Active Directory

### DISCUSSION THREAD

**Discussion by 19rellimcm37** [zitadel/zitadel#8729] [General]
Trying to integrate Zitadel with an Windows ec2 that is running Active Directory Role.  Both of them live in the same VPC. 

When trying to log in as Other User with a domain account, using an email address for the login, getting "User could not be found" error.  Don't see a way to look for errors from Zitadel.  Windows Event Viewer isn't showing any errors for the failed login.  

Under Organization > Identity Providers, the Identity Provider created has a green check for Availability

Have the following entries, please advise if they are correct or not: ``

Name - `domain_controller01`

Servers -` ldaps://domain_name.company.com:389 `

BaseDN - `DC=domain_controller01,DC=company.com`

BindDN - `company.com\user (also tried company.com\user.company.com)`

Userbase - `DC=domain_controller01,DC=company.com`

User filters - `(sAMAccountName=user)`

User Object Classes - `top; person; organizationalPerson; user `

ID attributes - `(userPrincipalName=user@company.com)`

**Comment by hifabienne** [comment]
@livio-a @muhlemmer can you help here?

**Reply by muhlemmer** [reply]
Sorry, not at the moment. I don't have a windows AD setup and at the moment it would take too much time to get a test setup up and running.
